test/integration/full/controls/big_query.rb

```diff
@@ -40,3 +41,17 @@
   its('time_partitioning.type') { should be nil }
   its('clustering') { should be nil }
 end
+
+describe google_bigquery_table(project: "#{project_id}", dataset: "#{dataset_name}", name: "#{external_tables[:csv_example][:friendly_name]}") do
+  it { should exist }
+  its('friendly_name') { should eq "#{external_tables[:csv_example][:friendly_name]}" }
+  its('time_partitioning.type') { should be nil }
+  its('clustering') { should be nil }
```

Comment:
In the tests, it would be good if we verified one or more of the additional settings (ex. field_delimiter).

Reply:
Good point - addressed this in 4e0de65